Tests/DangerTests/DangerDSLTests.swift

```diff
@@ -34,7 +34,7 @@ final class DangerDSLTests: XCTestCase {
         XCTAssertTrue(danger.runningOnGithub)
         XCTAssertTrue(danger.supportsSuggestions)
         XCTAssertNotNil(danger.git)
-        XCTAssert(danger.github.api.configuration.accessToken == "7bd263f8e4becaa3d29b25d534fe6d5f3b555ccf")
+        XCTAssert(danger.github.api.configuration.accessToken == "7bd263f8e4becaa3d29b25d534fe6d5f3b555ccf".data(using: .utf8)?.base64EncodedString())
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 14...
  
  
  DangerID: danger-id-Danger;
  File: Tests/DangerTests/DangerDSLTests.swift;
  Line: 37;
-->

- :warning: Line should be 140 characters or less: currently 150 characters (`line_length`)